docs/openshift.md

```diff
@@ -34,11 +34,11 @@ Following steps will allow you to deploy the Operator and Percona XtraDB Cluster
 
     !!! note
     
-        If you are going to install the Operator in [multi-namespace (cluster-wide) mode](cluster-wide.md), please choose values with `-cw` suffix for the update channel and version, and select the "All namespaces on the cluster" radio button for the installation mode instead of chosing a specific Namespace:
+        To install the Operator in [multi-namespace (cluster-wide) mode](cluster-wide.md), find it in the Certified catalogue. Choose values with `-cw` suffix for the update channel and version, and select the "All namespaces on the cluster" radio button for the installation mode instead of chosing a specific Namespace:
```

Comment:
```suggestion
        To install the Operator in [multi-namespace (cluster-wide) mode](cluster-wide.md), use the one with one the Certified label. Choose values with `-cw` suffix for the update channel and version, and select the "All namespaces on the cluster" radio button for the installation mode instead of chosing a specific Namespace:
```